.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "django-google-spanner",
+  "name_pretty": "Cloud Spanner Django",
+  "product_documentation": "https://cloud.google.com/spanner/docs/",
+  "client_documentation": "https://googleapis.dev/django/spanner/latest",
+  "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:190851%2B%20status:open",
+  "release_level": "ga",
+  "language": "python",
+  "library_type": "GAPIC_COMBO",
+  "repo": "googleapis/python-spanner-django",
+  "distribution_name": "django-google-spanner",
+  "api_id": "django-google-spanner.googleapis.com",
```

Comment:
This can be omitted since this library does not talk directly to a Google API endpoint

Reply:
removed `"api_id": "django-google-spanner.googleapis.com"`, please let me know if any other key also needs to be removed example: `"requires_billing": true` ?